src/vite-plugins/src/index.ts

```diff
@@ -139,3 +141,53 @@ export const replicaForwardPlugin = ({
     });
   },
 });
+
+/** Update the HTML files to include integrity hashes for script.
+ * i.e.: `<script src="foo.js">` becomes `<script integrity="<hash(./foo.js)>" src="foo.js">`.
+ */
+export const integrityPlugin: Plugin = {
```

Comment:
It's quite surprising to me that there is no ready-made plugin for that. Or do we have additional requirements to build our own?

Reply:
There is some, but they pull in an HTML parser and did not work for me

Reply:
Thanks for the clarification.